components/private/ThumbTrack/src/MDCThumbTrack.m

```diff
@@ -36,6 +37,26 @@
 static const CGFloat kValueLabelWidth = 0.81f * kValueLabelHeight;
 static const CGFloat kValueLabelFontSize = 12.f;
 
+static UIColor *MDCThumbTrackValueLabelTextColorDefault() {
+  return UIColor.whiteColor;
+}
+
+static UIColor *MDCThumbTrackValueLabelBackgroundColorDefault() {
+  return UIColor.blueColor;
+}
+
+static UIColor *MDCThumbTrackTrackOnColorDefault() {
+  return UIColor.blueColor;
+}
+
+static UIColor *MDCThumbTrackThumbEnabledColorDefault() {
+  return UIColor.blueColor;
+}
+
+static UIColor *MDCThumbTrackInkColorDefault() {
```

Comment:
Drop the `MDCThumbTrack` prefix for the API names because these aren't exported APIs.

Reply:
I've seen multiple instances where we keep the class prefix for static methods (and constants) in implementation files.  Is your request a shift in guidance for all components?

Reply:
This is due to some legacy internal requirements that are no longer the case, related to how we were building our .m files. We no longer need to prefix static APIs.